e2etests/web/regular_integration_tests/lib/treeshaking_main.dart

```diff
@@ -0,0 +1,45 @@
+
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      key: const Key('mainapp'),
+      title: 'Integration Test App',
+      home: MyHomePage(title: 'Integration Test App'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
```

Comment:
Do we care about treeshaking from user-code too? If so, we can add an override for `debugFillProperties` in this class, and make sure it's being treeshaken out.

Reply:
Same code path as other Widgets, decided not to add.